package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "segment-volume-lambda",
+  "version": "1.0.0",
+  "description": "Pushing volume data from segment into datadog",
+  "scripts": {
+    "depcheck": "./bin/depcheck.sh",
+    "dev": "serverless offline start --config serverless-dev.yml",
+    "process:volumes": "node ./dist/index.js",
+    "lint": "eslint .",
+    "serve": "serverless offline start --config serverless-dev.yml",
+    "test": "jest",
+    "test:integration": "jest integration -c jest.integration.config.js",
+    "typecheck": "tsc --noEmit"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/immediate-media/segment-volume-lambda.git"
+  },
+  "dependencies": {
+    "@datadog/datadog-api-client": "^1.22.0",
+    "axios": "^1.6.7",
+    "dotenv": "^16.4.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.5",
+    "@babel/plugin-transform-runtime": "^7.23.4",
+    "@babel/preset-env": "^7.23.5",
+    "@commitlint/config-conventional": "^12.1.4",
+    "@types/aws-lambda": "^8.10.130",
+    "@types/jest": "^29.5.10",
+    "@types/node-fetch": "^2.6.9",
+    "@types/original-url": "^1.2.3",
```

Comment:
Can get rid of `@types/original-url` and `@types/node-fetch` as we aren't using the related packages (being `original-url` and `node-fetch`).